payments/card/direct.md

```diff
@@ -0,0 +1,217 @@
+    - url: /payments/card/
+      title: Introduction
+    - url: /payments/card/redirect
+      title: Redirect
+    - url: /payments/card/seamless-view
+      title: Seamless View
+    - url: /payments/card/direct
+      title: Direct
+    - url: /payments/card/after-payment
+      title: After Payment
+    - url: /payments/card/other-features
+      title: Other Features
+---
+
+{% include alert.html type="warning"
+                      icon="warning"
+                      header="Section under review"
+                      body="This section of the Developer Portal is under
+                      review and may be incomplete or contain minor errors." %}
+
+{% include jumbotron.html body="The Direct Payment scenario **is used by
+customers that are compliant with PCI-DSS regulations**, and is a way to
+implement card payments without using Swedbank Pay hosted payment page." %}
+
+{% include alert.html type="danger" icon="error" header="PCI-DSS Complicance"
+body="The direct integration option requires you to collect the card data on
+your website, which means it must be [PCI-DSS Compliant](
+https://www.pcisecuritystandards.org/)." %}
+
+## Introduction
+
+* The payer places an order and you make a `POST` request towards Swedbank Pay
+  with gathered `Purchase` information.
+* The action taken next is the `direct-authorization` operation that is returned
+  in the first request. You `POST` the payer's card data to the URL in
+  the[`direct-authorization` operation][authorization].
+* If 3-D Secure authentication is required, you will then receive a URL where
+  you will have to redirect the payer.
+* When the payment is completed, the payer needs to be redirected back to your
+  merchant/webshop site.
+* Finally you make a `GET` request towards Swedbank Pay with the `paymentID`
+  received in the first step, which will return the purchase result.
+
+The sequence diagram below shows a high level description of a complete
+purchase, and the requests you have to send to Swedbank Pay. The links will take
+you directly to the corresponding API description.
+
+```mermaid
+sequenceDiagram
+    participant Payer
+    participant Merchant
+    participant SwedbankPay as Swedbank Pay
+
+    activate Payer
+    Payer->>+Merchant: start purchase
+    deactivate Payer
+    Merchant->>+SwedbankPay: POST /psp/creditcard/payments
+    deactivate Merchant
+    note left of Merchant: First API Request
+    SwedbankPay-->>+Merchant: rel: direct-authorization
+    deactivate SwedbankPay
+    Merchant->>+SwedbankPay: GET <payment.id>
+    deactivate Merchant
+    note left of Merchant: Second API request
+    SwedbankPay-->>+Merchant: rel: direct-authorization
+    deactivate SwedbankPay
+    Merchant-->>-Payer: display purchase result
+```
+
+## API Requests
+
+The API requests are displayed in the [purchase flow](#purchase-flow-mobile).
+You can [create a card `payment`][create-payment] with following `operation`
+options:
+
+* [Purchase][purchase]
```

Comment:
```suggestion
* [`Purchase`][purchase]
```
Isn't this and the other entries on this list refering to a code element, and should thus be marked as code elements?